src/controllers/API.js

```diff
@@ -185,9 +185,22 @@ class APIController {
     });
   }
 
-  static responseFromExternalError(error, requestAccepts) {
+  /**
+   * Builds a response from errors. Allows errors that occur outside of the
+   * library to be handled and returned in JSON API-compiant fashion.
+   *
+   * @param {Error|APIError|Error[]|APIError[]} errors Error or array of errors
+   * @param {string} requestAccepts Request's Accepts header
+   */
+  static responseFromExternalError(errors, requestAccepts) {
     let response = new Response();
-    response.errors = [APIError.fromError(error)];
+
+    // Convert to array
+    response.errors = Array.isArray(errors) ? errors : [errors];
+
+    // Convert Errors to APIErrors
+    response.errors = response.errors.map((error) => APIError.fromError(error));
```

Comment:
This is good. Out of cusiousity, though, why use an arrow function here (instead of just passing in `APIError.fromExternalError`? Is it for the `this`-binding inside the `fromExternalError` method? In that case, a simple `.bind()` call would work too/be shorter.